pow-migration/src/state/mod.rs

```diff
@@ -433,6 +445,14 @@ pub async fn get_stakers(
         }
     }
 
+    // Now that we have the stakers, update the validator's total stake
+    for staker in stakers.values() {
+        let validator = validators
+            .get_mut(&staker.delegation.to_string())
+            .expect("Delegated validator must exist");
+        validator.total_stake += staker.balance;
+    }
```

Comment:
A suggestion to make the `total_stake` consistent upon every time `get_stakers` being called.

```suggestion
    // Now that we have the stakers, recompute the validator's total stake
    for genesis_validator in validators.values_mut() {
        let stakers: Vec<&GenesisStaker> = stakers
            .iter()
            .filter_map(|staker| {
                if staker.1.delegation == genesis_validator.validator.validator_address {
                    return Some(staker.1);
                }
                None
            })
            .collect();

        genesis_validator.total_stake = Coin::from_u64_unchecked(Policy::VALIDATOR_DEPOSIT)
            + stakers
                .iter()
                .fold(Coin::ZERO, |acc, staker| acc + staker.balance);
    }
```